src/main/java/fi/metatavu/metaform/server/metaforms/FieldController.java

```diff
@@ -57,11 +59,23 @@ public FieldFilters parseFilters(Metaform metaform, List<String> filterList) {
       return null;
     }
     
-    List<FieldFilter> filters = filterList.stream()
+    List<String> filterListCombined = new ArrayList<>(filterList.size());
+    
+    filterList.stream()
       .filter(StringUtils::isNoneEmpty)
+      .forEach((filter) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1602))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=26&issues=AWPKM9Kx4427HkKVgpDi&open=AWPKM9Kx4427HkKVgpDi)

[ISSUE_KEY]: AWPKM9Kx4427HkKVgpDi